src/Bundle/Config/ConfigResolver.php

```diff
@@ -169,7 +169,7 @@ private function normalizeLoadingOrder(array $loadingOrder, array $replace): arr
         return $loadingOrder;
     }
 
-    private function replaceBundleNames(array &$loadAfter, array $replace): void
+    private function replaceBundleNames(array & $loadAfter, array $replace): void
```

Comment:
Did you do this on purpose or was it the CS fixer?

Reply:
That's part of [7addeac](https://github.com/contao/manager-plugin/pull/43/commits/7addeac912d4f5a155a469999c85dc3afd3512cb) which just contains automatic fixes of the cs fixer

Reply:
I ran `composer bin ecs update` and running the cs fixer again undo the changes ([db09287](https://github.com/contao/manager-plugin/pull/43/commits/db09287998ecf5f1a6fe783f6d0485c93cb28c0f)).